hphp/runtime/vm/jit/vasm-lower-inl.h

```diff
@@ -27,6 +27,28 @@ namespace HPHP { namespace jit {
 
 ///////////////////////////////////////////////////////////////////////////////
 
+namespace detail {
+
+/*
+ * Computes the dominating vreg constraint for each block. When lowering, this
+ * will be used to determine if a lowerer can use a new vreg as a scratch, or
+ * if it must rely on platform specific assembler scratch registers.
+ * The dominating restriction is used initially for the block and then updated
+ * whenever a vregrestrict/vregunrestrict is encountered.
+ *
+ *   INT_MAX: uninitialized
```

Comment:
This should not be part of the API—no consumer of `computeDominatingConstraints()`'s return should ever see any INT_MAX's.